colorbleed/__init__.py

```diff
@@ -3,6 +3,7 @@
 from avalon import api as avalon
 
 from .launcher_actions import register_launcher_actions
+from .manager_actions import register_manager_actions
```

Comment:
Let's see if we can drop this here. :) Since we don't need the global register_manager_actions - see other comment.